style.css

```diff
@@ -0,0 +1,47 @@
+body {
```

Comment:
@elysemclark Excellent job in this file. Right off the bat I see great indentation and empty lines between each section. Awesome!